ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -345,16 +358,14 @@ internal class CustomerCenterViewModelImpl(
     }
 
     @SuppressWarnings("ForbiddenComment")
-    override fun openURL(context: Context, url: Uri) {
-        context.openUriOrElse(url.toString()) { exception ->
-            val msg = if (exception is ActivityNotFoundException) {
-                context.getString(R.string.no_browser_cannot_open_link)
-            } else {
-                context.getString(R.string.cannot_open_link)
-            }
-            Toast.makeText(context, msg, Toast.LENGTH_SHORT).show()
-            Logger.w(msg)
+    override fun openURL(context: Context, url: Uri, method: CustomerCenterConfigData.HelpPath.OpenMethod) {
+        val openingMethod = when (method) {
+            CustomerCenterConfigData.HelpPath.OpenMethod.IN_APP -> URLOpeningMethod.IN_APP_BROWSER
+            CustomerCenterConfigData.HelpPath.OpenMethod.EXTERNAL,
+            CustomerCenterConfigData.HelpPath.OpenMethod.UNKNOWN,
+            -> URLOpeningMethod.EXTERNAL_BROWSER
         }
+        URLOpener.openURL(context, url.toString(), openingMethod)
```

Comment:
Hmm do we need to parse the url as an `Uri` at all?

Reply:
no, I changed it to String